lib/srv/app/server_test.go

```diff
@@ -664,9 +664,9 @@ func TestHandleConnectionWS(t *testing.T) {
 	s := SetUpSuiteWithConfig(t, suiteConfig{
 		ValidateRequest: func(s *Suite, r *http.Request) {
 			require.Equal(t, "on", r.Header.Get(common.XForwardedSSL))
-			// Websockets will pass on the server port at this level due to the
-			// websocket transport header rewriter delegate.
-			require.Equal(t, s.serverPort, r.Header.Get(forward.XForwardedPort))
+			// The port is not rewritten for WebSocket requests because it uses
+			// the same port as the original request.
+			require.Equal(t, "443", r.Header.Get(reverseproxy.XForwardedPort))
```

Comment:
I'm not sure if I got this change. Is this a behavior change? (Should we keep the previous behavior instead of changing it?). If it is not, why change it to a hardcoded port?

Reply:
It's not a behavior change. 
Previously, websockets used a different header rewriter that didn't change the XFP of the request.